.github/workflows/_build_test.yml

```diff
@@ -30,19 +29,42 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - run: ./build.sh ${{ matrix.component }}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
 
-      # TODO re-enable as soon as we make use of it
-      # - name: Scan Image
-      #   uses: ./.github/actions/scan-image
-      #   continue-on-error: true
-      #   with:
-      #     image_name: ${{ matrix.component }}
+      - name: Set up Docker Buildx
+        id: builder
+        uses: docker/setup-buildx-action@v3
+        with:
+          platforms: linux/amd64,linux/arm64
 
-      - uses: ./.github/actions/push-ghcr
+      - uses: docker/login-action@v3
         with:
-          docker_image: wikibase/${{ matrix.component }}
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # Current issue is generating an appropriate dev tag for the image
+      # I would like to have nx manage this through pre-release numbers,
+      # but may need to do something less complicated first to figure-out
+      # the target workflow.
+      #
+      # Multi platform builds are REALLY slow on Github Actions currently.
+      # We may need to use https://github.com/baschny/append-buildx-action
+      # and an external runner for the ARM64 builds, running either off of
+      # a local ARM machine on the team or a ARM-based VPS to get off qemu.
+      # To enable multi-platform builds add this to the buildx bake params:
+      #
+      # --set default.platform=linux/amd64,linux/arm64
+      #
+      # Timeout-minutes also need to be increased
+      - run: |
+          export $(grep -v '^#' build.env | xargs)
+          # export IMAGE_REGISTRY=ghcr.io
+          # export IMAGE_NAMESPACE=${{ github.repository_owner }}
+          export PR_BUILD_TAG=ghcr.io/${{ github.repository_owner }}/wikibase/${{ matrix.component }}:dev-${{ github.run_id }}
+          docker buildx bake --set default.tags=$PR_BUILD_TAG --push
```

Comment:
I intend to replace this with `nx build <image-name> --set ... --push` once I get a couple more things sorted on that API, but this is fine for now